src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -492,4 +492,40 @@ object StringUtils {
         val clean = removeColor()
         return "§$firstColor§m$clean"
     }
+
+    fun getListOfStringsMatchingLastWord(words: Array<String>, args: Collection<String>): List<String> {
+        val lastWord = words.lastOrNull() ?: return emptyList()
+        val matches = args.filter { it.startsWith(lastWord, ignoreCase = true) }
+        return matches.ifEmpty {
```

Comment:
what's the point of this? i think this can go

Reply:
for tab completion